README.md

```diff
@@ -3,14 +3,15 @@
 ## salesman
 **See**: [demo](https://lovasoa.github.io/salesman.js/)  
 **Author:** Ophir LOJKINE
+
 salesman npm module
 
 Good heuristic for the traveling salesman problem using simulated annealing.  
 
 * [salesman](#module_salesman)
     * [~Point](#module_salesman..Point)
         * [new Point(x, y)](#new_module_salesman..Point_new)
-    * [~solve(points, [temp_coeff], [callback=])](#module_salesman..solve) ⇒ <code>Array.&lt;Number&gt;</code>
+    * [~solve(points, [temp_coeff], [callback], [callback])](#module_salesman..solve) ⇒ <code>[ &#x27;Array&#x27; ].&lt;Number&gt;</code>
```

Comment:
Why `['Array']` ?

Reply:
I didn't do that, jsdoc-to-markdown created that file.